src/components/UriViewer.js

```diff
@@ -56,9 +56,14 @@ const isFilePreviewable = ({ size, ...metadata }) => {
   return !isBinary(metadata) && (isText(metadata) || (isImage(metadata) && size <= 1e9))
 }
 
-const isGs = uri => _.startsWith('gs://', uri)
+export const isGs = uri => _.startsWith('gs://', uri)
 
-const isDrs = uri => _.startsWith('dos://', uri) || _.startsWith('drs://', uri)
+export const isDrs = uri => _.startsWith('dos://', uri) || _.startsWith('drs://', uri)
+
+export const isAzureUri = uri => {
+  const azureRegex = RegExp('^https://(.+).blob.core.windows.net')
+  return azureRegex.test(uri)
+}
```

Comment:
nit: move this `const azureRegex` out of `isAzureUri()` and up to the root level of `UriViewer.js` so we're not re-creating a new `RegExp` for every invocation of `isAzureUri()`

```suggestion
const azureRegex = RegExp('^https://(.+).blob.core.windows.net')

export const isAzureUri = uri => {
  return uri.match(azureRegex)
}
```

note that I've also changed from `RegExp.test()` to `String.match()` to avoid problems with regex reuse: https://blog.rebased.pl/2021/03/16/a-regular-expression-surprise-in-javascript.html (we don't use the /g flag so it wouldn't hit us, but still)

Reply:
great point here -- updated!